scripts/processTranslations.js

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs')
+const i18next = require('i18next')
+
+const localizedFiles = fs.readdirSync('locales')
+const resources = localizedFiles.reduce((acc, file) => {
+  const lng = file.substring(0, file.length - '.json'.length)
+  return {
+    ...acc,
+    [lng]: { translation: require(`../locales/${lng}.json`) },
```

Comment:
isn't this:

```
[lng]: { translation: require(`../locales/${file}`) },
```

(using `lng` to reconstruct `file` made me think there was something else going on)